src/graphql/schema/uuid/taxonomy-term/resolvers.ts

```diff
@@ -36,13 +37,21 @@ export const resolvers: TaxonomyTermResolvers = {
         id: taxonomyTerm.parentId,
       })
     },
-    async children(taxonomyTerm, _args, { dataSources }) {
+    async children(taxonomyTerm, { ...cursorPayload }, { dataSources }) {
```

Comment:
This can just be `cursorPayload`:
```suggestion
    async children(taxonomyTerm, cursorPayload, { dataSources }) {
```